splash/examples/proxyEg.lua

```diff
@@ -0,0 +1,19 @@
+function main(splash)
```

Comment:
I think "use-proxy.lua" would be a better name (the convention in the `examples/` folder is to use dash-separated file name, not camel case).